python_loops_py.ipynb

```diff
@@ -0,0 +1,411 @@
+        "id": "eYFejqdx57ZX",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "dff8798c-4408-4446-f476-30d0c5fec1b7"
+      },
+      "execution_count": 96,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Type your name: laura\n",
+            "l lima\n",
+            "a alfa\n",
+            "u uniform\n",
+            "r romeo\n",
+            "a alfa\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "b= []\n",
+        "a=[[1,2,3],[4,5,6],[7,8,9]]\n",
+        "for x in a:\n",
+        "  print(x)\n",
+        "  for y in x:\n",
+        "    b.append(y)\n",
+        "print(b)\n",
+        "c=[]\n",
+        "d=[]\n",
+        "e=[]\n",
+        "for z in b:\n",
+        "  print(z)\n",
+        "  if z%3==1:\n",
+        "    c.append(z)\n",
+        "  elif z%3==2:\n",
+        "    d.append(z)\n",
+        "  elif z%3==0:\n",
+        "    e.append(z)\n",
+        "f=[c,d,e]\n",
+        "print(f)\n",
+        "    \n",
+        "\n",
+        "\n",
+        "    \n",
+        "    \n",
+        "\n",
+        " "
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "IQ4f2fCV4Ozw",
+        "outputId": "ceed5bf5-854e-4466-d891-4c549dceef37"
+      },
+      "execution_count": 118,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "[1, 2, 3]\n",
+            "[4, 5, 6]\n",
+            "[7, 8, 9]\n",
+            "[1, 2, 3, 4, 5, 6, 7, 8, 9]\n",
+            "1\n",
+            "2\n",
+            "3\n",
+            "4\n",
+            "5\n",
+            "6\n",
+            "7\n",
+            "8\n",
+            "9\n",
+            "[[1, 4, 7], [2, 5, 8], [3, 6, 9]]\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "for x in range (0,5):\n",
+        "  print(x)"
+      ],
+      "metadata": {
+        "id": "hMTdyQ7ONN9R"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "a=[[1,2,3],[4,5,6],[7,8,9]]\n",
+        "clen = []\n",
+        "for i in range (0,3):\n",
+        "  for j in range (0,3):\n",
+        "    clen.append(a[i][j])\n",
+        "  b.append(clen)\n",
+        "  clen=[]\n",
+        "print(b)\n"
+      ],
+      "metadata": {
+        "id": "Q2Nv6d-MP5C0"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "flat_list = []\n",
+        "list = ['whatever',['1','2','3'],'3er']\n",
+        "\n",
+        "for member in list:\n",
+        "    if type(member) is list:\n",
+        "        for item in member:\n",
+        "            flat_list.append(item)\n",
```

Comment:
So creating a new list from the original instead of using `break` and appending to the original list. :x: